### ISSUE TITLE
Is F4 still a failure?

### ISSUE THREAD

**Issue by dd8** [w3c/wcag#1184]
If an element references a style with `text-decoration: blink` F4 triggers a failure of SC 2.2.2
https://www.w3.org/WAI/WCAG21/Techniques/failures/F4.html

But no current browsers support  `text-decoration: blink`
https://developer.mozilla.org/en-US/docs/Web/CSS/text-decoration#Browser_compatibility

The last browsers to support `text-decoration: blink` were:
- Firefox 23 released in August 2013 (support removed in FF 24 release in September 2013)
- Opera in 2013 (support removed when Opera switched to Chromium engine in May 2013)

The CSS 2.1 recommendation suggested that conforming user agents should not support blink:
>Text blinks (alternates between visible and invisible). Conforming user agents may simply not blink the text. Note that not blinking the text is one technique to satisfy checkpoint 3.3 of WAI-UAAG.

https://www.w3.org/TR/CSS2/text.html#lining-striking-props

There are still ways to simulate blink using CSS animations:
https://gist.github.com/mugifly/5099109

**Comment by patrickhlauke** [comment]
i'd say add a note to F4 with a condensed explanation that in practice this is not a failure anymore, but the example could be kept around for historical reasons? also, not sure i ever think of failure techniques "triggering" a failure, more that they're common examples/exemplars of something that fails a particular SC...but perhaps that's splitting semantic hairs.

**Comment by dd8** [comment]
I wasn't 100% sure about Opera Mini, but seems like it's never supported `blink` (or any other animation) since it pre-renders on server:
https://dev.opera.com/articles/opera-mini-content-authoring-guidelines/#image-animation-video

**Comment by patrickhlauke** [comment]
Wondering if this can be solved by marking the technique as obsolete, or if it's better to just outright delete it.

**Comment by fstrr** [comment]
[F4 and F47 will be updated in the next couple of weeks](https://github.com/w3c/wcag/pull/4166) to be marked as obsolete, and be replaced by a technique based on CSS blinking.

Based on that, this looks like it can be closed.
